locales/ro/LC_MESSAGES/volto.po

```diff
@@ -0,0 +1,1939 @@
+msgid "Add block..."
+msgstr "Adăugați bloc ..."
+
+#: components/manage/Blocks/Listing/QuerystringWidget
+msgid "Add criteria"
+msgstr "Adăugați criterii"
+
+#: components/manage/Widgets/SchemaWidget
+msgid "Add field"
+msgstr "Adăugați câmp"
+
+#: components/manage/Widgets/SchemaWidget
+msgid "Add fieldset"
+msgstr "Adăugare set de câmpuri"
+
+#: components/manage/Controlpanels/UsersControlpanel
+msgid "Add group"
+msgstr "Adăugare grup"
+
+#: components/manage/Controlpanels/ContentTypes
+msgid "Add new content type"
+msgstr "Adăugați un nou tip de conținut"
+
+#: components/manage/Controlpanels/UsersControlpanel
+msgid "Add new group"
+msgstr "Adăugare grup nou"
+
+#: components/manage/Controlpanels/UsersControlpanel
+msgid "Add new user"
+msgstr "Adăugare utilizator nou"
+
+#: components/manage/Controlpanels/UsersControlpanel
+msgid "Add to Groups"
+msgstr "Adăugare la grupuri"
+
+#: components/manage/Add/Add
+msgid "Add {type}"
+msgstr "Adăugare {tip}"
+
+#: components/manage/Controlpanels/AddonsControlpanel
+msgid "Add-ons Settings"
+msgstr "Setări add-on-uri"
+
+#: components/manage/Blocks/Image/ImageSidebar
+#: components/manage/Blocks/LeadImage/LeadImageSidebar
+#: components/manage/Blocks/Maps/MapsSidebar
+#: components/manage/Blocks/Video/VideoSidebar
+msgid "Alignment"
+msgstr "Aliniere"
+
+#: components/manage/Contents/Contents
+msgid "All"
+msgstr "Toate"
+
+#: components/manage/Blocks/Image/ImageSidebar
+#: components/manage/Blocks/LeadImage/LeadImageSidebar
+msgid "Alt text"
+msgstr "Alt text"
+
+#: components/manage/Widgets/SchemaWidget
+msgid "Are you sure you want to delete this field?"
+msgstr "Sunteți sigur că doriți să ștergeți acest câmp?"
+
+#: components/manage/Widgets/SchemaWidget
+msgid "Are you sure you want to delete this fieldset including all fields?"
+msgstr "Sunteți sigur că doriți să ștergeți acest set de câmpuri, inclusiv toate câmpurile?"
+
+#: components/manage/Contents/Contents
+msgid "Ascending"
+msgstr "Ascendent"
+
+#: components/manage/Controlpanels/AddonsControlpanel
+msgid "Available"
+msgstr "Disponibil"
+
+#: components/manage/Contents/Contents
+#: components/manage/Controlpanels/AddonsControlpanel
+#: components/manage/Controlpanels/ContentType
+#: components/manage/Controlpanels/ContentTypes
+#: components/manage/Controlpanels/Controlpanel
+#: components/manage/Controlpanels/Controlpanels
+#: components/manage/Controlpanels/ModerateComments
+#: components/manage/Controlpanels/UsersControlpanel
+#: components/manage/Diff/Diff
+#: components/manage/History/History
+#: components/manage/Preferences/ChangePassword
+#: components/manage/Preferences/PersonalInformation
+#: components/manage/Preferences/PersonalPreferences
+#: components/manage/Sharing/Sharing
+#: components/manage/Toolbar/Toolbar
+#: components/theme/ContactForm/ContactForm
+msgid "Back"
+msgstr "Înapoi"
+
+#: components/manage/Diff/Diff
+msgid "Base"
+msgstr "Baza"
+
+#: components/manage/Sidebar/Sidebar
+msgid "Block"
+msgstr "Blocare"
```

Comment:
It's the object `Bloc` not the action :wink:

Reply:
Fixed.